app/views/content/events/get-the-most-from-events/_content.html.erb

```diff
@@ -110,6 +110,19 @@
       <h3>Event staff</h3>
       <p> Our friendly team will be there to welcome you at the door. Ask them about the event, where to find the experts, and where you can get overseas qualifications checked.</p>
 
+    <h3>What a Get Into Teaching Event will be like</h3>
```

Comment:
```suggestion
    <h3>What a Get Into Teaching event will be like</h3>
```

Reply:
Is there another way we could phrase this h2? We've just told them what it would be like - what extra are they getting from the video?

(If you don't think it adds to the page, happy to discuss!)